app/src/main/java/org/mozilla/focus/browser/integration/NavigationButtonsIntegration.kt

```diff
@@ -29,11 +30,16 @@ class NavigationButtonsIntegration(
 ) : LifecycleAwareFeature {
     private var scope: CoroutineScope? = null
 
-    private val enabledColorRes = context.theme.resolveAttribute(R.attr.primaryText)
-    private val disabledColorRes = context.theme.resolveAttribute(R.attr.disabled)
+    private var enabledColorRes = context.theme.resolveAttribute(R.attr.primaryText)
+    private var disabledColorRes = context.theme.resolveAttribute(R.attr.disabled)
 
     init {
 
+        if (store.state.findCustomTabOrSelectedTab(customTabId)?.ifCustomTab() != null) {
+            enabledColorRes = R.color.enabled_toolbar_button_color
+            disabledColorRes = R.color.disabled_toolbar_button_color
+        }
```

Comment:
Custom tabs can have different toolbar colors, so we need to pick the tint color dynamically. For the text we do:
https://github.com/mozilla-mobile/android-components/blob/main/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt#L114